packages/amplication-code-gen-types/src/schemas/decimalNumber.json

```diff
@@ -6,6 +6,11 @@
   "additionalProperties": false,
   "required": ["minimumValue", "maximumValue", "precision"],
   "properties": {
+    "dataType": {
+      "type": "string",
+      "enum": ["DECIMAL", "FLOAT"],
```

Comment:
it might be worth adding a description that state that this is going to be the Prisma `model` field scalar types.
I'm assuming that is the purpose of this.